app/init/database/filters.php

```diff
@@ -71,18 +71,27 @@ function (mixed $value) {
     },
     function (mixed $value, Document $document, Database $database) {
         $attributes = $database->find('attributes', [
-            Query::equal('collectionInternalId', [$document->getInternalId()]),
+            Query::equal('collectionInternalId', [$document->getSequence()]),
             Query::equal('databaseInternalId', [$document->getAttribute('databaseInternalId')]),
             Query::limit($database->getLimitForAttributes()),
         ]);
 
         foreach ($attributes as $attribute) {
-            if ($attribute->getAttribute('type') === Database::VAR_RELATIONSHIP) {
-                $options = $attribute->getAttribute('options');
-                foreach ($options as $key => $value) {
-                    $attribute->setAttribute($key, $value);
-                }
-                $attribute->removeAttribute('options');
+            $attributeType = $attribute->getAttribute('type');
+
+            switch ($attributeType) {
+                case Database::VAR_RELATIONSHIP:
+                    $options = $attribute->getAttribute('options');
+                    foreach ($options as $key => $value) {
+                        $attribute->setAttribute($key, $value);
+                    }
+                    $attribute->removeAttribute('options');
+                    break;
+
+                case Database::VAR_STRING:
+                    $filters = $attribute->getAttribute('filters', []);
+                    $attribute->setAttribute('encrypt', in_array('encrypt', $filters));
```

Comment:
Why do we set this "virtual" attribute is this just a convenience or we have other reasons for it?